tempesta_fw/str.c

```diff
@@ -340,12 +349,12 @@ tfw_str_eq_cstr(const TfwStr *str, const char *cstr, int cstr_len,
 			       ? strncasecmp
 			       : strncmp;
 
-	BUG_ON(str->len && !str->ptr);
+	BUG_ON(str->len && !str->chunks);
 	TFW_STR_FOR_EACH_CHUNK(chunk, str, end) {
-		BUG_ON(chunk->len &&  !chunk->ptr);
-
+		BUG_ON(chunk->len &&  !chunk->data);
+		TFW_DBG("eq_cstr:chl:%lu;sl:%d\n", chunk->len, clen);
```

Comment:
The debug print seems useful only in your development process and could be useless for others, please remove such prints